src/js/libcs/Json.js

```diff
@@ -16,7 +16,7 @@ Json.turnIntoCSJson = function (a) {
 };
 
 Json._helper.ShallowClone = function (o) {
-    var out, v, key;
+    let out, v, key;
     out = Array.isArray(o) ? [] : {};
     for (key in o) {
```

Comment:
Change `for (key in o)` to `for (const key in o)` and remove `key` from `let` statement above.